flume-ng-doc/sphinx/FlumeUserGuide.rst

```diff
@@ -3920,7 +3920,7 @@ Property Name          Default      Description
 **type**               --           The component type name has to be ``remove_header``
 withName               --           Name of the header to remove
 fromList               --           List of headers to remove, separated with the separator specified with from.list.separator
-fromListSeparator    \s*,\s*      Regular expression used to separate multiple header names in the list specified using from.list
+fromListSeparator      \s*,\s*      Regular expression used to separate multiple header names in the list specified using from.list
```

Comment:
`\\s*,\\s*` should be in the default column, `\s*,\s*` will be translated to `s*,s*`. (Double backslash is needed to display a single one)